portality/static/doaj/js/available_facetviews/admin.editorgroups.facetview.js

```diff
@@ -21,7 +21,7 @@ jQuery(document).ready(function($) {
             {'display':'Created Date','field':'created_date'},
             {'display':'Last Modified Date','field':'last_updated'},
             {'display':'Editor ID','field':'editor'},
```

Comment:
@richard-jones I don't like it. Adding ```.exact``` to the search fields means you must match the letter case when you are searching, e.g. to get "West Africa" you can't type 'west' or 'west africa'.

Also, if I add the ```.exact``` to the editor ID sort field, the one editor who's name begins with an upper-case character is at the top of the list, before 'a'. Without the ```.exact``` they are in the correct place.